x-pack/plugins/actions/server/index.ts

```diff
@@ -51,103 +50,6 @@ export const config: PluginConfigDescriptor<ActionsConfig> = {
   exposeToBrowser: {
     email: { domain_allowlist: true },
   },
-  deprecations: ({ renameFromRoot, unused }) => [
```

Comment:
Just curiosity as I worked on something similar recently, why is removing this ok? Can't these config options still pop up in some old `kibana.yml`?

Reply:
The typical flow is to mark things as deprecated for some amount of time, or until some telemetry tells us it's no longer in use, and then we can remove it.  I believe this is scheduled to be in 9.x, so customers have had prolly all of 8.x to "fix" these.

Or maybe I'm misunderstanding your question ...

Reply:
No no, that was it thank you.

> ... for some amount of time, or until some telemetry tells us it's no longer in use,

this in particular 😁